pkgs/development/libraries/libfpx/default.nix

```diff
@@ -11,6 +11,13 @@ stdenv.mkDerivation rec {
   # Darwin gets misdetected as Windows without this
   NIX_CFLAGS_COMPILE = if stdenv.isDarwin then "-D__unix" else null;
 
+  patches = [
+    (fetchpatch {
+      url = "https://gitweb.gentoo.org/repo/gentoo.git/plain/media-libs/libfpx/files/libfpx-1.3.1_p6-gcc6.patch";
```

Comment:
Please refer to a commit, otherwise the url will break if gentoo removes the patch.